client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/provisioning/index.tsx

```diff
@@ -41,7 +41,9 @@ export const Provisioning: FC< ProvisioningProps > = ( { status } ) => {
 	// Error handler.
 	if ( currentAction.status === 'error' ) {
 		const contactClickHandler = () => {
-			recordTracksEvent( 'calypso_onboarding_site_migration_instructions_error_contact_support' );
+			recordTracksEvent( 'calypso_site_migration_instructions_link_click', {
+				linkname: 'error_contact_support',
```

Comment:
We should use dashes here as we do for the `go-to-plugin-page` and `install-plugin` values.

Reply:
Good catch! https://github.com/Automattic/wp-calypso/pull/92845/commits/c1fdb5bd0e1ba2aaaa7b11d82f27e5856a787ea0